infrastructure/main.tf

```diff
@@ -82,10 +82,8 @@ module "bulk-scan" {
     S2S_NAME   = "${var.s2s_name}"
     S2S_SECRET = "${data.azurerm_key_vault_secret.s2s_secret.value}"
 
-    REUPLOAD_BATCH     = "${var.reupload_batch}"
-    REUPLOAD_DELAY     = "${var.reupload_delay}"
     REUPLOAD_MAX_TRIES = "${var.reupload_max_tries}"
```

Comment:
this one stays, I will rename it separately (to match the job name)

Reply:
it is not used at all - can be removed without regret